weather_mv/loader_pipeline/pipeline.py

```diff
@@ -54,9 +54,7 @@ def pipeline(known_args: argparse.Namespace, pipeline_args: t.List[str]) -> None
     known_args.first_uri = next(iter(all_uris))
 
     with beam.Pipeline(argv=pipeline_args) as p:
-        if known_args.zarr:
-            paths = p
```

Comment:
Isn't this needed? How do we select a single Zarr URI?

Reply:
For above query following URI Patterns will work for `zarr`.
- `../test_data/test_data.zarr` (For a single zarr)
- `../test_data/*.zarr` (For multiple zarr in a directory)

Reply:
This will be handled in `else` part of this condition.
Currently it's working for a single `zarr` file.

But for URI like `gs://gcp-public-data-arco-era5/co/model-level-*.zarr` it's not running as intended.

Reply:
Thanks for the ptr! That makes sense to me. 

For that test Zarr: I'm happy to meet over VC to debug what's going on.